patches/server/0962-Piglin-Dancing-API.patch

```diff
@@ -26,7 +26,7 @@ index 48d0a4e42e1b90d1323784d1284acabfe9497dd6..b3b0d58082d39575788d93ba7c787576
 +
 +    @Override
 +    public void setDancing(int duration) {
```

Comment:
Maybe change the type to `long`? iirc the `setMemoryWithExpiry` method also accepts a `long` value